src/microbe_stage/gui/ExtinctionBox.tscn

```diff
@@ -25,21 +25,18 @@ tracks/0/keys = {
 "values": [ Color( 1, 1, 1, 0 ), Color( 1, 1, 1, 1 ) ]
 }
 
-[node name="ExtinctionBox" type="Control"]
+[node name="ExtinctionBox" type="Popup"]
+visible = true
```

Comment:
Shouldn't popup be invisible by default? Doesn't this show that yellow warning triangle in Godot editor?